### ISSUE TITLE
[FEATURE] [RFC] Create a new "Permissions Evaluator" UI when setting up a Role

### ISSUE THREAD

**Issue by derek-ho** [opensearch-project/security-dashboards-plugin#1590]
**Is your feature request related to a problem?**
Connected to https://github.com/opensearch-project/security/issues/3381, when setting up a role, I would like to see what permissions this role needs to perform a certain set of actions. 

**What solution would you like?**

**What alternatives have you considered?**

**Do you have any additional context?**

**Comment by davidlago** [comment]
[Triaging] Thanks for filing, @derek-ho! This one requires a bit more detailed UX before we can call it triaged.

**Comment by scrawfor99** [comment]
This issue is not feasible for the time being so going to close. Please open a new issue if additional information can be added.
